cmd/analyse.go

```diff
@@ -6,18 +6,22 @@ import (
 
 	"github.com/fabric8-analytics/cli-tools/analyses/driver"
 	sa "github.com/fabric8-analytics/cli-tools/analyses/stackanalyses"
+	"github.com/fatih/color"
 	"github.com/rs/zerolog/log"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
 var manifestFile string
+var jsonOut bool
+var flagNoColor bool
 
 // analyseCmd represents the analyse command
 var analyseCmd = &cobra.Command{
-	Use:     "analyse",
-	Short:   "Performs full Stack Analyses on CRDA Platform.",
-	Long:    `Performs full Stack Analyses on CRDA Platform. Supported ecosystems are Pypi (Python), Maven (Java), Npm (Node) and Golang (Go).`,
+	Use:   "analyse",
+	Short: "Performs full Stack Analyses on CRDA Platform.",
```

Comment:
Provides detailed report of vulnerabilities.

Reply:
I dont think you need to add CRDA platform.

Reply:
Ack